### ISSUE TITLE
Gerbera 1.9.2 Daemon active(running) but UI not available

### ISSUE THREAD

**Issue by severnaside** [gerbera/gerbera#2420]
The Web UI could not be reached.  I checked the status of gerbera - see output below:

● gerbera.service - Gerbera Media Server
     Loaded: loaded (/usr/lib/systemd/system/gerbera.service; enabled; vendor preset: disabled)
     Active: active (running) since Thu 2022-02-10 12:27:21 GMT; 2min 38s ago
   Main PID: 497 (gerbera)
      Tasks: 42 (limit: 16669)
     Memory: 55.8M
        CPU: 338ms
     CGroup: /system.slice/gerbera.service
             └─497 /usr/bin/gerbera -c /home/gerbera/.config/gerbera/config.xml

Feb 10 12:27:21 itx1 gerbera[497]: 2022-02-10 12:27:21   info: Loading autoscan location: /home/gerbera/Music; recursive: true, last_modified: 2021-06-27 21:36:06
Feb 10 12:27:21 itx1 gerbera[497]: 2022-02-10 12:27:21   info: Initialising UPnP with interface: <unset>, port: <unset>
Feb 10 12:27:21 itx1 gerbera[497]: [*** LOG ERROR #0001 ***] [2022-02-10 12:27:21] [] {string pointer is null}
Feb 10 12:27:22 itx1 gerbera[497]: [*** LOG ERROR #0002 ***] [2022-02-10 12:27:22] [] {string pointer is null}
Feb 10 12:27:24 itx1 gerbera[497]: [*** LOG ERROR #0003 ***] [2022-02-10 12:27:24] [] {string pointer is null}
Feb 10 12:27:27 itx1 gerbera[497]: [*** LOG ERROR #0004 ***] [2022-02-10 12:27:27] [] {string pointer is null}
Feb 10 12:27:31 itx1 gerbera[497]: 2022-02-10 12:27:31   info: IPv4: Server bound to: 192.168.1.12:49152
Feb 10 12:27:31 itx1 gerbera[497]: 2022-02-10 12:27:31   info: IPv6: Server bound to: :49152
Feb 10 12:27:31 itx1 gerbera[497]: 2022-02-10 12:27:31   info: IPv6 ULA/GLA: Server bound to: :49153
Feb 10 12:27:31 itx1 gerbera[497]: 2022-02-10 12:27:31   info: The Web UI can be reached by following this link: http://192.168.1.12:49152/

I realise this may be similar to Issue #1599.  Anyway I solved my problem by inserting the following line into the gerbera.service file:

`ExecStartPre=/bin/sh -c 'until ping -c1 192.168.1.1; do sleep 1; done;'`

There my router is pinged.  Alternatively one could ping a website eg google.com

OS: Archlinux 64-bit
gerbera version: 1.9.2

**Comment by KarlStraussberger** [comment]
You have to ensure that the your unit file contains sufficient precondition that your network is up before gerbera tries to detect the network interface:
```
After=mysql.target network-online.target
```

The network target on your OS may differ which may be the reason for your startup issue

**Comment by severnaside** [comment]
Thank you @KarlStraussberger - you put me on the right track.  I had to do something different because I am using ConnMan, not NetworkManager:

`After=connman-wait-online.service`

The connman-wait-online.service has to be enabled, but it would not stay enabled after a reboot.  I had to change the connman-wait-online.service [Install] section from:
`WantedBy=network-online.target`
to:
`WantedBy=multi-user.target`

I will close this issue.
